test/end-to-end.js

```diff
@@ -324,14 +324,14 @@ describe('end-to-end', function() {
     });
   });
 
-  describe('hello-world template', function() {
+  describe('notes template', function() {
```

Comment:
I'd like to have both `notes` and `hello-world` templates covered by these tests.